fe/fe-core/src/main/java/com/starrocks/sql/ast/InsertStmt.java

```diff
@@ -277,9 +277,14 @@ public Table makeTableFunctionTable() {
         checkState(tableFunctionAsTargetTable, "tableFunctionAsTargetTable is false");
         // fetch schema from query
         QueryRelation query = getQueryStatement().getQueryRelation();
-        List<Field> allFields = query.getRelationFields().getAllFields();
-        List<Column> columns = allFields.stream().filter(Field::isVisible).map(field -> new Column(field.getName(),
-                field.getType(), field.isNullable())).collect(Collectors.toList());
+        List<Column> columns = query.getRelationFields().getAllFields().stream()
+                .filter(Field::isVisible)
+                .map(field -> new Column(field.getName(), field.getType(), field.isNullable()))
+                .collect(Collectors.toList());
+
+        if (columns.stream().map(Column::getName).distinct().count() != columns.size()) {
```

Comment:
you'd better to find what's the duplicate column and then return error msg to user client

Reply:
done